packages/commonwealth/client/scripts/views/components/CommunityInformationForm/CommunityInformationForm.tsx

```diff
@@ -265,7 +196,7 @@ const BasicInformationForm = ({
         enableGenerativeAI
       />
 
-      {socialLinksDisplay ? (
+      {withSocialLinks ? (
```

Comment:
this comment is not to this line but to image upload. According to the ticket, image shuold be optional but so far we require it in community creation flow. My point is that we should make the same validation for both flows

Reply:
yeah its actually required (we have no community without an image) - so I kept it required.